### ISSUE TITLE
v1.5.10 broke ctrl+z with Dvorak keyboard layout

### ISSUE THREAD

**Issue by nyurik** [kasecato/vscode-intellij-idea-keybindings#307]
This only happened in v1.5.10. Switching back to v1.5.9 fixes this issue.

To reproduce (on Linux), install this plugin in VSCode and configure your system to use Dvorak keyboard layout.  In a typescript file in VSCode, make several different changes, and use "undo" shortcut -- `Ctrl+/` **multiple times** (the `/` key is mapped to `z` in Dvorak).

Expected: multiple undos
Actual: current line gets commented and un-commented with the `//`, as if I was using QWERTY

### Versions
```
Version: 1.81.1
Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794
Date: 2023-08-09T22:18:39.991Z
Electron: 22.3.18
ElectronBuildId: 22689846
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 5.15.0-79-generic

---

Name: IntelliJ IDEA Keybindings
Id: k--kato.intellij-idea-keybindings
Description: Port of IntelliJ IDEA Keybindings, including for WebStorm, PyCharm, PHP Storm, etc.
Version: 1.5.10
Publisher: Keisuke Kato
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=k--kato.intellij-idea-keybindings
```

Possibly related issues
* https://github.com/microsoft/vscode/issues/173447
* https://github.com/microsoft/vscode/issues/173325

**Comment by nyurik** [comment]
I am not sure this issue belongs to the vscode repo, but I think authors of https://github.com/microsoft/vscode/pull/173456 half a year ago might have some insight into how an extension could disrupt their fix.  CC: @hamzahamidi and @alexdima

**Comment by kasecato** [comment]
Looks like this should use a virtual key code

### As-Is
```json
            {
                "key": "ctrl+[Slash]",
                "mac": "cmd+[Slash]",
                "command": "editor.action.commentLine",
                "when": "editorTextFocus && !editorReadonly",
                "intellij": "Comment/uncomment with line comment"
            },
```

### To-Be
```json
            {
                "key": "ctrl+oem_2",
                "mac": "cmd+[Slash]",
                "command": "editor.action.commentLine",
                "when": "editorTextFocus && !editorReadonly",
                "intellij": "Comment/uncomment with line comment"
            },
```

- https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-layouts
- https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-layoutindependent-bindings
- #294
- #305

**Comment by nyurik** [comment]
Thanks for looking into it!  Does the `"mac"` section need to be updated too? (I don't use Mac, so no idea, but others might hit this bug there as well)

**Comment by kasecato** [comment]
That's true
It would be better to fire key codes according to the keyboard layout in all operating systems
Let me revert #305 since they should change the keyboard layout in thier OS
```json
            {
                "key": "ctrl+/",
                "mac": "cmd+/",
                "command": "editor.action.commentLine",
                "when": "editorTextFocus && !editorReadonly",
                "intellij": "Comment/uncomment with line comment"
            },
```

**Comment by kasecato** [comment]
just released v1.7.0 that supports for Dvorak layouts
If there're issues with the layout, we will guide users to change your layout in their own OS
